views/pages/RegistrationPage/RegistrationPage.test.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { screen, render } from '@testing-library/react'
+import { MemoryRouter } from 'react-router-dom'
+import RegistrationPage from '.'
+import { NotificationContext } from '../../contexts'
+
+describe('Registration Page', () => {
+  test('renders the page', () => {
+    render(
+      <NotificationContext.Provider value={[{}, jest.fn()]}>
+        <RegistrationPage />
+      </NotificationContext.Provider>,
+      { wrapper: MemoryRouter }
+    )
+
+    const pageTitle = screen.getByText('Sign up')
+    const signInLink = screen.getByText('Sign in')
+    const formInputs = screen.getAllByRole('textbox')
+
+    expect(pageTitle).toBeInTheDocument()
+    expect(signInLink).toHaveAttribute('href')
```

Comment:
can we add the href value here?

```suggestion
    expect(signInLink).toHaveAttribute('href', routes.signIn)
```